lib/timeline/.eslintrc

```diff
@@ -19,20 +19,7 @@
     "no-unreachable": 1,
     "no-useless-escape": 0,
     "no-console": 0,
-    "require-jsdoc": ["error", {
```

Comment:
This has nothing to do with the subgroups and should better be done is a seperate RP.